spec/factories/wishlists.rb

```diff
@@ -3,5 +3,12 @@
     name "Birthday"
     description "Tom's 18th birthday"
     user
+
+    factory :wishlist_with_items do
```

Comment:
If you are throwing this away you can also change "Gift1" to "Gift". I had mentors requested this kind of changes before.